spec/factories/batches.rb

```diff
@@ -32,7 +32,10 @@
     vaccine
 
     name { "#{prefix}#{Faker::Number.number(digits: 4)}" }
-    expiry { Faker::Time.forward(days: 50, period: :morning) }
+
+    trait :not_expired do
```

Comment:
Is this `not_expired` or maybe `with_expiry`? Just thinking that `not_expired` sounds like a batch that has expired in the past, but I believe this is setting an expiry date in the future.

Reply:
Actually, ignore that. It suddenly makes more sense with the `expired` trait below.

Reply:
Yeah, I did wonder about `with_expiry`, but then `expired` is also `with_expiry`. I might swap them around though in the file, I think it's more natural for expired to come first.